tests/README.md

```diff
@@ -0,0 +1,107 @@
+            - https://moodle.webserver:8000/
+            - https://moodle.webserver:8000/admin/oauth2callback.php
+         origins: []
+         application_type: "web"
+      
+      # start ocis
+      PROXY_ENABLE_BASIC_AUTH=true OCIS_INSECURE=true PROXY_HTTP_ADDR=0.0.0.0:9200 OCIS_URL=https://host.docker.internal:9200 PROXY_TRANSPORT_TLS_KEY=ocis.pem PROXY_TRANSPORT_TLS_CERT=ocis.crt ./ocis/bin/ocis server
+   ```
+   To run Behat tests you have to add some code in moodle source code for ssl proxy support 
+   ```bash
+       sed -i "s/\$CFG->dataroot = \$CFG->behat_dataroot;/\$CFG->dataroot = \$CFG->behat_dataroot;\n\t\t\$CFG->sslproxy = \$CFG->behat_sslproxy;/" /<path-to-moodle>/lib/setup.php
+   ```
+   configure traefik and test enviroments
+   ```bash
+      cd moodle-docker
+      mkdir -p traefik/configs traefik/certificates
+      cd traefik/configs
+      cat >> tls.yml <<'EOF'
+      tls:
+        stores:
+           default:
+             defaultCertificate:
+                certFile: /certificates/server.crt
+                keyFile: /certificates/server.key
+      EOF
+      
+      # some general settings for moodle
+      export MOODLE_DOCKER_WWWROOT=<path-of-your-moodle-source-code>
+      export MOODLE_DOCKER_DB=pgsql
+      export MOODLE_DOCKER_PHP_VERSION=8.1
+      export MOODLE_DOCKER_WEB_HOST=host.docker.internal
+      export MOODLE_DOCKER_WEB_PORT=8000
+      export MOODLE_DOCKER_SELENIUM_VNC_PORT=5900
+      export MOODLE_DOCKER_BROWSER=chrome:latest
+      
+      cp config.docker-template.php $MOODLE_DOCKER_WWWROOT/config.php
+      cd ../moodle
+   ```
+
+   change in `config.php` for ssl support
+   ```bash
+      sed -i "s/\$CFG->wwwroot\s*=\s*\"http:\/\/{\$host}\";/\$CFG->sslproxy = true;\n\$CFG->wwwroot = \"https:\/\/{\$host}\";/" $MOODLE_DOCKER_WWWROOT/config.php
+      sed -i "s/\$CFG->behat_wwwroot\s*=\s*'http:\/\/webserver';/\$CFG->behat_sslproxy = true;\n\$CFG->behat_wwwroot = 'https:\/\/moodle.webserver:8000';/" $MOODLE_DOCKER_WWWROOT/config.php
+   ```
+   ```php
+      $CFG->behat_profiles = array(
+          'default' => array(
+             'browser' => getenv('MOODLE_DOCKER_BROWSER'),
+             'wd_host' => 'http://selenium:4444/wd/hub',
+          ),
+      );
+      # replace above code in config.php with below code
+      $CFG->behat_profiles = array(
+        'default' => array(
+        'browser' => getenv('MOODLE_DOCKER_BROWSER'),
+        'wd_host' => 'http://selenium:4444/wd/hub',
+        'capabilities' =>
+            [
+                'acceptSslCerts'=> true,
+                'extra_capabilities' =>
+                    [ 'chromeOptions' =>
+                        [ 'args' =>
+                            [ '--ignore-certificate-errors']
+                        ]
+                    ]
+            ]
+        ),
+      );
+   ```
+   configure the test container
+   ```bash
+      cd ..
+      cd moodle-docker
```

Comment:
I think we don't need to change the folders, but can stay inside of `moodle-docker` all the time